### ISSUE TITLE
Need better UI indication to inform users about blocking unsafe scripts on web page

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#5562]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Currently we show a secure connection padlock for pages with unsafe scripts. The notification is only shown when clicked on the padlock in URL bar.  Some sites require the unsafe scripts to be loaded to render properly. Showing a secure connection for such sites might lead users to think Brave is not compatible with such sites. We need to have a better indication when user visits such sites.

**Expected behavior:**
Need better UI indication to inform users about blocking unsafe scripts on web page. One option is to use a red [unlocked padlock ](http://fontawesome.io/icon/unlock-alt/)to indicate unsafe scripts on page. A clear UI notification should be shown upon visiting such sites similar to Opera/Chrome

- Platform (Win7, 8, 10? macOS? Linux distro?): 
All

- Brave Version: 
0.12.9 Preview 1

- Screenshot if needed:
Brave
![image](https://cloud.githubusercontent.com/assets/17010094/20218109/d5ce49f8-a81a-11e6-99da-dd2def3a61de.png)

  Chrome
  ![image](https://cloud.githubusercontent.com/assets/17010094/20217931/2ff922e6-a81a-11e6-973d-fca8d548648b.png)

  Firefox
  ![image](https://cloud.githubusercontent.com/assets/17010094/20217960/4f9e0db4-a81a-11e6-8165-d89aa0219b29.png)

  Opera
  ![image](https://cloud.githubusercontent.com/assets/17010094/20218002/71eb8a04-a81a-11e6-8b74-6fd28f2677fc.png)

  cc: @bradleyrichter

**Comment by privatzee** [comment]
Good point. I'd add this: whatever the situation, always have Brave clearly tell which option is **recommended** so that the average user knows what to choose (at least at first) without having to read anything more. 99% will never read more, once Brave is in general use
